lib/network/CachedImage.js

```diff
@@ -12,7 +12,7 @@
  */
 class CachedImage {
   /**
-   * @inheritDoc
+   * @ignore
```

Comment:
Neither `@inheritdoc` nor `@ignore` does anything useful in the output for the constructor. The output is the same as if they were not present. Can they both be removed please? They don't add anything useful.